api/backend_test.go

```diff
@@ -382,3 +383,35 @@ func TestStartStopMultipleTimes(t *testing.T) {
 	require.NoError(t, backend.StartNode(config))
 	require.NoError(t, backend.StopNode())
 }
+
+func TestHashMessage(t *testing.T) {
+	backend := NewStatusBackend()
+	config, err := utils.MakeTestNodeConfig(params.StatusChainNetworkID)
+	require.NoError(t, err)
+	err = backend.StartNode(config)
+	require.NoError(t, err)
+	defer func() {
+		require.NoError(t, backend.StopNode())
+	}()
+
+	key, err := crypto.GenerateKey()
+	require.NoError(t, err)
+	addr := crypto.PubkeyToAddress(key.PublicKey)
+
+	originalMessage := []byte{0x01, 0x02, 0x03}
+	hash := backend.HashMessage(originalMessage)
+
+	// simulate signature from external signer like a keycard
+	sig, err := crypto.Sign(hash, key)
+	require.NoError(t, err)
+	sig[64] += 27
```

Comment:
This definitely needs an explanation :P

Reply:
yeah it's true, shall I add a comment pointing to https://github.com/ethereum/go-ethereum/blob/master/internal/ethapi/api.go#L431 ?